src/main/java/commands/AddUnmark.java

```diff
@@ -0,0 +1,52 @@
+package commands;
+
+import exceptions.EmptyDescriptionException;
+import exceptions.TooManyParametersException;
+import windebot.History;
+import windebot.Reminder;
+import windebot.Ui;
+
+/**
+ * The AddUnmark class represents a command to unmark a task as done in the reminder list.
+ * This command unmarks a specified task by its index in the list.
+ * If the input is invalid or the task cannot be found, an exception is thrown.
+ */
+
+public class AddUnmark extends Command {
```

Comment:
Looks great! Perhaps you can consider changing the class name to reflect that a task is being 'unmarked' instead of using 'Add'.